tests/server/check_server.c

```diff
@@ -23,7 +23,9 @@ static UA_Server *server = NULL;
 static void setup(void) {
     server = UA_Server_new();
     ck_assert(server != NULL);
-    UA_ServerConfig_setDefault(UA_Server_getConfig(server));
+    UA_ServerConfig *config = UA_Server_getConfig(server);
```

Comment:
@jpfr Because the REUSEADDR option is disabled by default, some tests fail with an 'address already in use' message.
My suggestion is to activate the REUSEADDR option for all tests. What do you think?

Reply:
I don't see why a test should fail.
Do we spin up parallel servers with the same port?
Then we should change the port for one of these...